src/arrays/ladderops.jl

```diff
@@ -22,10 +22,60 @@
     before_eye(lens, pivot) = speye(prod(lens[[1:pivot-1]]))
     after_eye(lens, pivot) = speye(prod(lens[[pivot+1:length(lens)]]))
     function eye_sandwich(lens, pivot, op)
-        return kron(before_eye(lens, pivot), 
-                    op, 
+        return kron(before_eye(lens, pivot),
+                    op,
                     after_eye(lens, pivot))
     end
 
+function positionop(n::Int)
+    cop = raiseop(n)
+    return scale!(1/sqrt(2.),cop+cop')
+end
+
+function momentumop(n::Int)
+    cop = raiseop(n)
+    return scale(im/sqrt(2.), cop-cop')
+end
+
+function spin_h1(j::Float64)
```

Comment:
Maybe it is more elegant to use `Rational`s here (see [docs](http://julia.readthedocs.org/en/latest/manual/complex-and-rational-numbers/#rational-numbers))?